portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+import javax.servlet.ServletContextListener;
+import javax.servlet.ServletContextEvent;
+import com.googlecode.objectify.ObjectifyService;
+
+public class YourBootstrapper implements ServletContextListener {
```

Comment:
Since it's not obvious from this code what it is, please leave some javadoc class level comments.